greet.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
```

Comment:
**TODO FOR LOST POINTS**

Just a top level comment that your work should be in a ``lab-caleb`` folder per the instructions.